cmd/main.go

```diff
@@ -209,3 +218,21 @@ func triggerInterval(intervalChannel chan bool, productionInterval time.Duration
 	time.Sleep(productionInterval)
 	intervalChannel <- true
 }
+
+func addPeertoDiscoveryList(ip, port string) {
+	client := http.Client{}
+	//send initally and then wait every 5 minutes
+	r, err := requests.AddPeerToDiscoveryRequest(ip, port)
+	if err != nil {
+		log.Fatalf("Failed to make Add Peer request: %s", err.Error())
+	}
+	client.Do(r)
+	for range time.Tick(time.Second * 300) { //5 minutes
+		r, err := requests.AddPeerToDiscoveryRequest(ip, port)
+		if err != nil {
+			log.Fatalf("Failed to make Add Peer request: %s", err.Error())
+		}
+		client.Do(r)
```

Comment:
Can we handle the `Do` error and just log it if it fails? Not fatally but just log it in case the peer server goes down.

Reply:
Done